src/rt/dmain2.d

```diff
@@ -324,6 +328,21 @@ extern (C) int _d_run_main(int argc, char **argv, MainFunc mainFunc)
             fldcw   fpucw;
         }
     }
+    version (DragonFlyBSD) version (D_InlineAsm_X86)
```

Comment:
I thought you're not supporting 32-bit?  Take this out.

Reply:
Correct :+1: